ex03.ex

```diff
@@ -101,7 +112,9 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  def list_equal([ a | b ], [ a | b ]), do: true
```

Comment:
## -4

Two deductions. First, a bug, as if doesn't handle empty lists.

Second, it doesn't really do anything, because

~~~ elixir
def list_equal([ a | b ], [ a | b ]), do: true
~~~

is the same as saying:

~~~ elixir
def list_equal([ a1 | b1 ], [ a2 | b2 ]) 
when a1 == a2 and b1 == b2, do: true
~~~

and so you're using == to compare two lists (b1 == b2)